### ISSUE TITLE
Add an option for Litematica Schematics under edit instance like how worlds and screenshots are

### ISSUE THREAD

**Issue by Mylezz** [MultiMC/Launcher#4729]
### Role

For Litematica Schematics

### Suggestion

Add a tab for Schematics like how you can edit per instance worlds and screenshots

### Benefit

Ease of adding Schematics to specific profiles 

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

Enough people use Litematica that this could be beneficial for everyone and it's simple enough to implement it would add to Multimc's feature set and make it a more robust program.

**Comment by ErrorBot1122** [comment]
YES, but I'm not sure that's really practical. The only way I could see this work is by adding a program to edit schematics as well, then I would make sense.
